docs/general/performance.md

```diff
@@ -109,6 +109,26 @@ new Chart(ctx, {
 });
 ```
 
+### Enable spanGaps
+
+If you have a lot of data points, it can be more performant to enable spanGaps. This disables segmentation of line, which can be an unneeded step.
```

Comment:
Should spanGaps be enclosed in backticks? I'd probably say "segmentation of the line"

Reply:
done